SK.WebApp/package.json

```diff
@@ -16,6 +16,7 @@
     "@reduxjs/toolkit": "^1.5.0",
     "atomic-layout": "^0.16.2",
     "date-fns": "^2.17.0",
+    "fetch": "^1.1.0",
```

Comment:
There's no need for that package for now. You're even not using it in the code 😄 Please remove that. Fetch comes standard with every modern browser

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API

Reply:
I removed this